src/SFA.DAS.Funding.SystemAcceptanceTests/StepDefinitions/Fm36StepDefinitions.cs

```diff
@@ -47,6 +48,34 @@ public async Task GetFm36Data(TokenisableDateTime searchDate)
 
     }
 
+    [Given(@"that there is at least 15 records available from FM36 endpoint")]
+    public async Task GivenThatRecordsExistInFm36Endpoint()
+    {
+        // The perpose of this endpoint is to ensure paging tests can be run. There should be
```

Comment:
I wouldn't pull up a comment typo on it's own but since I have requested other changes may as well fix the type in "purpose"